packages/main/src/SegmentedButton.hbs

```diff
@@ -2,6 +2,11 @@
 	@click="{{_onclick}}"
 	@focusin="{{_onfocusin}}"
 	class="ui5-segmentedbutton-root"
+	role="radiogroup"
 >
-	<slot></slot>
+	{{#each this.buttons}}
+		<div class="ui5-segmentedbutton-item" role="radio" aria-checked="{{this.pressed}}">

```

Comment:
1. The class seems to do nothing? Unless there should be some css for it, please delete it

2. `aria-checked` does not change upon space. It changes whenever the segmented button itself is rerendered later, for whatever reason. If you want the segmented button to be invalidated whenever the children are invalidated (and you do in this case), you should use `listenFor` in the slot metadata.

`"default": {
			propertyName: "buttons",
			type: HTMLElement,
			individualSlots: true,
			listenFor: { include: ["pressed"] }
		},`
Once you do this, whenever `pressed` changes for a toggle button, the template for segmented button will be re-evaluated and the aria-checked binding will be updated as well.

Reply:
Done. The classes are now currently used for selecting the wrapper divs and applying the style.